tests/foreman/cli/test_activationkey.py

```diff
@@ -1731,3 +1732,63 @@ def test_positive_ak_with_custom_product_on_rhel6(module_org, rhel6_contenthost,
     rhel6_contenthost.install_katello_ca(target_sat)
     result = rhel6_contenthost.register_contenthost(module_org.label, activation_key=ak.name)
     assert 'The system has been registered with ID' in result.stdout
+
+
+@pytest.mark.tier3
+def test_positive_invalid_release_version(module_sca_manifest_org, module_target_sat):
+    """Check invalid release versions when updating or creating an activation key
+
+    :id: 2efe452f-132c-4831-abfb-62305832ac66
+
+    :customerscenario: true
+
+    :Steps:
+        1. Attempt to create an activation key with a invalid release version
+        2. Attempt to update an activation key with a invalid release version
+        3. Successfully update an activation key with a valid release version
+
+    :expectedresults: Invalid release versions should fail and valid release version should succeed
+
+    :BZ: 1895976
+    """
+
+    name = gen_string('alpha')
+    activation_key = module_target_sat.cli_factory.make_activation_key(
+        {'organization-id': module_sca_manifest_org.id, 'name': name}
+    )
+    rh_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
+        basearch=DEFAULT_ARCHITECTURE,
+        org_id=module_sca_manifest_org.id,
+        product=REPOS['kickstart']['rhel8_aps']['product'],
+        repo=REPOS['kickstart']['rhel8_aps']['name'],
+        reposet=REPOS['kickstart']['rhel8_aps']['reposet'],
+        releasever=REPOS['kickstart']['rhel8_aps']['version'],
+    )
+    Repository.synchronize({'id': rh_repo_id})
```

Comment:
Can't you remove the import for this and just do 
```suggestion
    module_target_sat.cli.Repository.synchronize({'id': rh_repo_id})
```